net/unbound/Makefile

```diff
@@ -45,6 +45,16 @@ define Package/unbound/description
   This package contains the Unbound daemon.
 endef
 
+define Package/unbound/postinst
+#!/bin/sh
+
+group_exists unbound || group_add_next unbound
+
+gid=$$(grep -s '^unbound:' "$${IPKG_INSTROOT}/etc/group" | cut -d: -f3)
+
+user_exists unbound || user_add unbound '' "$$gid"
```

Comment:
Done. I decided on using 63 as the UID/GID as it now has to be hardcoded. 53 is already taken. I couldn't find any list for managing these and resorted to `grep`.